docs/source/model_design/general_research.rst

```diff
@@ -679,11 +679,131 @@ Conducting Systematic Literature Searches/Reviews
+
+However, DAGs are limited in that they require *prior causal knowledge* about the variables that may be causally related to a given exposure and outcome of interest. Therefore, there may be uncertain, unmeasured, or unknown variables that cannot be represented in a given DAG. Further, DAGs do not allow for easy representation of direction or magnitude of a causal relationship or for effect modification.
+
+For the purpose of this section of our documentation, we will assume that each DAG is completely accurate, represents **all** relevant variables in a given causal network and that there are no unrepresented variables.
+
+Types of Pathways Within Directed Acyclic Graphs
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+A path in a DAG is defined as any route drawn through arrows that link two variables **regardless of the direction of the arrows**.
+
+Directed Paths
+~~~~~~~~~~~~~~
+
+Directed paths are those in which variables are connected through arrows that flow all flow in the same direction. See below for an example of a directed path between variables A and D.
+
+.. image:: directional_dag.svg
+
+**In the case of a directed path between two variables, those variables will be causally related.**
+
+Undirected Paths
+~~~~~~~~~~~~~~~~
+
+Undirected paths are those in which variables are connected through arrows that do not all flow from head to tail. The two cases of undirected paths covered here include backdoor paths and colliders.
+
+**In the case that there is an undirected path (and no directed path) bewteen two variables, those variables will NOT be causally related, although they may be associated in some way.**
+
+Backdoor Paths
+^^^^^^^^^^^^^^
+
+Backdoor paths are those in which two variables are connected via a common cause. In other words, two variables that are linked via a variable that has two arrows flowing *away* from it. See below for an example in which variables Y and Z are connected via a backdoor path through variable X.
+
+.. image:: backdoor_path_dag.svg
+
+**In the case of a backdoor path between two variables (and no directional path), there will be an association (due to a common cause between them), but no causal relationship between the two variables.**
+
+Colliders
+^^^^^^^^^
+
+Colliders are variables that have more than one cause; in other words, where pathways collide and a variable has two arrows flowing *toward* it. See below for an example in which variable Z is a collider on the path between X and Y.
+
+.. image:: collider_dag.svg
+
+**In the case that two variables are linked through a pathway with a collider (and there is no directional or backdoor pathway between them), there will be no association or causal realtionship between the two variables.** This is also referred to as a **"closed path."**
```

Comment:
Again, consistency, and clarify what "this" refers to (please correct my interpretation if I got it wrong):

```suggestion
**In the case that two variables are linked through a pathway with a collider (and there is no directed or backdoor path between them), there will be no association or causal realtionship between the two variables.** A path with a collider is also referred to as a **"closed path."**
```

Reply:
Also, I think it would be good to mention that conditioning on a collider introduces dependence between its parents. See e.g.

https://en.wikipedia.org/wiki/Collider_(statistics)
https://en.wikipedia.org/wiki/Berkson%27s_paradox

This topic is probably out of scope for this PR, and perhaps should go in a separate section, but maybe add a Todo.

Reply:
Also it might be good to clarify that when we say that "a path has a collider" and hence is closed, we mean that the path contains the collider *and* its parents (i.e. we are abusing terminology by using the word "collider" to refer either to the single vertex Z or to the 3-vertex structure X -> Z <- Y, depending on context)...

For example, a path could still be open if it contains Z but does not contain X and Y.

Reply:
In fact, I think it might be a good idea to create a new subsection called "Open and Closed Paths" between thee "Colliders" section and the "Summary." You could move that final sentence defining open and closed paths to the new section, and clarify exactly what we mean when we say that a collider is or is not on a path.